command/troubleshoot/upstreams/troubleshoot_upstreams.go

```diff
@@ -74,14 +75,21 @@ func (c *cmd) Run(args []string) int {
 		return 1
 	}
 
+	c.UI.Output("==== Upstreams (explicit upstreams only)")
```

Comment:
I think with this output, to make sure it matches consul-k8s should be:

`==> Upstreams (explicit upstreams only) (count)` 

The same goes with Upstream IPs, but the count in the header.

Reply:
Sounds good,  will change it (I was following the format in RFC).